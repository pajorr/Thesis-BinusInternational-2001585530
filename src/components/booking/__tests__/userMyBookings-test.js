import React from 'react';
import { shallow } from 'enzyme';

import MyBooking from '../userMyBookings';


describe('userLogin', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<MyBooking debug/>);

        expect(component).toMatchSnapshot();
    });
});
