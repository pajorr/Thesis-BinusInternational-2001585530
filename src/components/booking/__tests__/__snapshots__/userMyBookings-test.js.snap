// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userLogin should render correctly in "debug" mode 1`] = `
<MyBooking
  classes={
    Object {
      "listStyle": "MyBooking-listStyle-2",
      "root": "MyBooking-root-1",
    }
  }
  debug={true}
/>
`;
